crates/ruff_linter/src/rules/pyupgrade/snapshots/ruff_linter__rules__pyupgrade__tests__UP013.py.snap

```diff
@@ -267,12 +267,13 @@ UP013.py:46:1: UP013 [*] Convert `X` from `TypedDict` functional to class syntax
    |
    = help: Convert `X` to class syntax
 
-ℹ Unsafe fix
+ℹ Safe fix
 43 43 | MyType = TypedDict("MyType", dict())
 44 44 | 
 45 45 | # Unsafe fix if comments are present
 46    |-X = TypedDict("X", {
 47    |-    "some_config": int,  # important
 48    |-})
-   46 |+class X(TypedDict):
-   47 |+    some_config: int
+   46 |+# important
+   47 |+class X(TypedDict):
+   48 |+    some_config: int
```

Comment:
This is a fantastic improvement, and we shouldn't let the perfect be the enemy of the good, so I'd happily merge a change to our fix infrastructure that preserved comments like this. However... I'd still mark this fix as unsafe if it moved comments like this. It would be very annoying if you e.g. had a TypedDict like this:

```py
Config = TypedDict("X", {
    "x": int,  # this is the x field. It's an int because numbers are great.
    "y": str,  # this is the y field. It's a string because some things are string.
    "z": bytes,  # this is the z field. TODO: why is it bytes??
})
```

And Ruff then autofixed it to this:

```py
# this is the x field. It's an int because numbers are great.
# this is the y field. It's a string because some things are string.
# this is the z field. TODO: why is it bytes??
class Config(TypedDict):
    x: int
    y: str
    z: bytes
```

One way to think about it is: would you be happy for Ruff to automatically apply these fixes every time you pressed save in an editor, like you might with an autoformatter?